WordPress/gutenberg
# Global style revisions: redesign style revision items

## What?

Resolves https://github.com/WordPress/gutenberg/issues/55647

This PR:

- shows humantimediff for changes < 1 day. Other dates are displayed as longer formats
- redesigns timeline and revision elements


https://github.com/WordPress/gutenberg/assets/6458278/6c59ef5e-d347-4c0c-abec-b179b672e23e


## Testing Instructions
Now open the styles revisions panel.

Check that the revision items work and the active hover states are correct.
